distribution/docs/src/main/resources/content/_tables/platform.ui.config.adoc

```diff
@@ -27,50 +27,57 @@
 |systemUsageTitle
 |String
 |A title for the system usage Message when the application is opened
-|true
 |
+|false
 
 |System Usage Message
 |systemUsageMessage
 |String
 |A system usage message to be displayed to the user each time the user opens the application
-|true
 |
+|false
 
 |Show System Usage Message once per session
 |systemUsageOncePerSession
-|With this selected, the system usage message will be shown once for each browser session. Uncheck this to have the usage message appear every time the search window is opened or refreshed.
 |Boolean
+|With this selected, the system usage message will be shown once for each browser session. Uncheck this to have the usage message appear every time the search window is opened or refreshed.
 |true
 |true
 
 |Header
 |header
-|Specifies the header text to be rendered on all pages.
 |String
-|true
+|Specifies the header text to be rendered on all pages.
```

Comment:
✏️ This column could be updated to consistently always end with periods or not end with periods.